client/js/views/visualize.react.jsx

```diff
@@ -103,7 +99,7 @@ var Visualize = React.createClass({
 
           <Row className='show-grid'>
             <Col xs={12} md={12}>
-              <Playbar currentCommit={this.state.commits[this.state.commitIndex]} numberOfCommits={this.state.commits.length-1} commitIndex={this.state.commitIndex} updateCommitIndex={this.updateCommitIndex} reset={this.reset}/>
```

Comment:
I don't know what reset is but I think I used the HEAD so if I accidentally deleted reset and the key below lmk